dist/constants.json

```diff
@@ -7825,697 +7825,697 @@
         "value": "673GH",
-        "label": "LECANTO VA CLINIC (TAMPA)"
+        "label": "Lecanto VA Clinic"
       },
       {
         "value": "516",
-        "label": "C.W. BILL YOUNG DEPT OF VAMC"
+        "label": "C.W. Bill Young Department of Veterans Affairs Medical Center"
       },
       {
         "value": "673",
-        "label": "JAMES A. HALEY VETERANS HOSP"
+        "label": "James A. Haley Veterans' Hospital"
       },
       {
         "value": "546",
-        "label": "BRUCE W. CARTER DEPT OF VAMC"
+        "label": "Bruce W. Carter Department of Veterans Affairs Medical Center"
       },
       {
         "value": "548",
-        "label": "WEST PALM BEACH VAMC"
+        "label": "West Palm Beach VA Medical Center"
       },
       {
         "value": "573",
-        "label": "MALCOM RANDALL DEPT OF VAMC"
+        "label": "Malcom Randall Department of Veterans Affairs Medical Center"
       },
       {
         "value": "675",
-        "label": "ORLANDO VAMC"
+        "label": "Orlando VA Medical Center"
       }
     ],
     "NY": [
       {
         "value": "528A4",
-        "label": "VA HEALTHCARE NETWORK UPSTATE NEW YORK SYSTEM VISN 2 - BATAVIA DIVISION"
+        "label": "Batavia VA Medical Center"
       },
       {
         "value": "528A5",
-        "label": "CANANDAIGUA VA MEDICAL CENTER"
+        "label": "Canandaigua VA Medical Center"
       },
       {
         "value": "528A6",
-        "label": "BATH VA MEDICAL CENTER"
+        "label": "Bath VA Medical Center"
       },
       {
         "value": "528A7",
-        "label": "SYRACUSE VA MEDICAL CENTER"
+        "label": "Syracuse VA Medical Center"
       },
       {
         "value": "528A8",
-        "label": "SAMUEL S. STRATTON VAMC"
+        "label": "Samuel S. Stratton Department of Veterans Affairs Medical Center"
       },
       {
         "value": "528GB",
-        "label": "JAMESTOWN VA CLINIC"
+        "label": "Jamestown VA Clinic"
       },
       {
         "value": "528GC",
-        "label": "DUNKIRK VA CLINIC"
+        "label": "Dunkirk VA Clinic"
       },
       {
         "value": "528GD",
-        "label": "NIAGARA FALLS VA CLINIC"
+        "label": "Niagara Falls VA Clinic"
       },
       {
         "value": "528GK",
-        "label": "LOCKPORT"
+        "label": "Lockport VA Clinic"
       },
       {
         "value": "528GQ",
-        "label": "LACKAWANNA CBOC"
+        "label": "Lackawanna VA Clinic"
       },
       {
         "value": "528GR",
-        "label": "OLEAN CBOC"
+        "label": "Olean VA Clinic"
       },
       {
         "value": "526GA",
-        "label": "WHITE PLAINS CLINIC"
+        "label": "White Plains VA Clinic"
       },
       {
         "value": "526GB",
-        "label": "YONKERS CLINIC"
+        "label": "Yonkers VA Clinic"
       },
       {
         "value": "526GD",
-        "label": "NORTH QUEENS CLINIC"
+        "label": "Thomas P. Noonan Jr. Department of Veterans Affairs Outpatient Clinic"
```

Comment:
- Consider abbreviated label for "Department of Veterans Affairs"
- Let's confirm this name change.